modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/ModuleProcess.cs

```diff
@@ -134,6 +137,40 @@ public class ModuleProcess : IProcessControl {
             }
         }
 
+        ///// <summary>
+        ///// Checks and warns about deprecated params and environment variables.
+        ///// </summary>
+        ///// <param name="logger"></param>
+        ///// <returns></returns>
+        private void CheckDeprecatedParams(ILogger logger) {
+            // List with pairs of deprecated and new/replacement options.
+            // If newOption is null, warning will not suggest using it instead.
+            var deprecatedOptions = new List<(string deprecatedOption, string newOption)> {
+                // Deprecated on 2020-09-17.
+                (LegacyCliConfigKeys.MaxOutgressMessages,
+                 LegacyCliConfigKeys.MaxEgressMessageQueue),
+
+                // Deprecated on 2020-09-17.
+                (PcsVariable.PCS_DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES,
```

Comment:
Both `PCS_DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES` and `PCS_DEFAULT_PUBLISH_MAX_EGRESS_MESSAGE_QUEUE` are part of backend OPC Publisher **Microservice** configuration. As far as I understand, none of them should be used for configuring OPC Publisher **Module**. I don't see them being used in module either. Can you please validate this and remove this case from the check here if they are not used ?

Reply:
I have removed DEFAULT_PUBLISH_MAX_OUTGRESS_MESSAGES. Will look for a better place (as part of another PR) or create a PBI to handle all deprecated PCS, as discussed.